src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1064,7 +1064,7 @@ namespace AppInstaller::CLI::Workflow
             }
 
             auto promptString = m_isApply ? Resource::String::ConfigurationWarningPromptApply : Resource::String::ConfigurationWarningPromptTest;
-            if (!context.Reporter.PromptForBoolResponse(promptString, Reporter::Level::Warning))
+            if (!context.Reporter.PromptForBoolResponse(promptString, Reporter::Level::Warning, true))

```

Comment:
```suggestion
            if (!context.Reporter.PromptForBoolResponse(promptString, Reporter::Level::Warning))

```

Let's make it default to false. Device configuration is quite a big change to the current device. We do want user confirm before execution. Plus we provided the ConfigurationAcceptWarning arg like AcceptPackageAgreements